### PR TITLE
Bump: Tomcat version to 9.0.88

### PR DESCRIPTION
**Pull request by BLasan**
## Purpose
- fixes: https://github.com/wso2-enterprise/wso2-apim-internal/issues/6143

### CODE REVIEW

File: tomcat-catalina-ha/9.0.88.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```
Comment (npamudika) [inline tomcat-catalina-ha/9.0.88.wso2v1/pom.xml:3]: Please update the year

File: tomcat-el-api/9.0.88.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```
Comment (npamudika) [inline tomcat-el-api/9.0.88.wso2v1/pom.xml:3]: Please update the year

File: tomcat-servlet-api/9.0.88.wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```
Comment (npamudika) [inline tomcat-servlet-api/9.0.88.wso2v1/pom.xml:3]: Please update the year

File: tomcat/9.0.88.wso2v1/pom.xml [hunk 4]
```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```
Comment (npamudika) [inline tomcat/9.0.88.wso2v1/pom.xml:3]: Please update the year

**Comment by BLasan** [comment]
@npamudika Please review

**Review by npamudika — APPROVED** [review]

**Pull request merged**